docs/content/mongocxx-v3/polyfill-selection.md

```diff
@@ -0,0 +1,59 @@
++++
+date = "2024-02-20T00:00:00-00:00"
+title = "Choosing a C++17 Polyfill"
+[menu.main]
+  identifier = "mongocxx3-polyfill-selection"
+  parent = "mongocxx3"
+  weight = 8
++++
```

Comment:
I must admit I do not understand how the value of some of these fields are supposed to be set (date and weight in particular), so simply copied the template from similar files.